src/__tests__/__actions__/resetPasswordActions.test.js

```diff
@@ -56,7 +59,7 @@ describe("ResetPassword action creators", () => {
         }
       ];
 
-      moxios.stubRequest(`${DEV_BASE_URL}/users/reset_password`, {
+      moxios.stubRequest(`${BASE_URL}/users/reset_password`, {
```

Comment:
@espoirMur, why this again? like changing all variable name. By the way, we are currently managing BASE_URL as env variable. Please rebase to get the change, and update the way you are doing this.

Reply:
Sorry about that...
I fixed it in my previous commit...